libraries/AP_InertialSensor/AP_InertialSensor_Backend.h

```diff
@@ -292,6 +296,12 @@ class AP_InertialSensor_Backend
     void update_accel(uint8_t instance) __RAMFUNC__; /* front end */
     void update_accel_filters(uint8_t instance) __RAMFUNC__; /* front end */
 
+    // catch updates to the primary gyro and accel
+    void update_primary_gyro() __RAMFUNC__; /* backend */
+    void update_primary_accel() __RAMFUNC__; /* backend */
+    virtual void set_primary_gyro(bool is_primary) {}
+    virtual void set_primary_accel(bool is_primary) {}
```

Comment:
can remove all the accel based switching

Reply:
Need to keep the set* for drivers that have different FIFOs and callbacks for accel vs gyro